src/checkout/components/PaymentsDetails/PaymentsDetails.tsx

```diff
@@ -0,0 +1,30 @@
+import { Grid, Input, Typography, Radio, useTranslate } from "../../../_shared";
+
+export const PaymentsDetails = () => {
+  const translate = useTranslate();
+
+  return (
+    <Grid container columnSpacing="18px" rowSpacing="24px">
+      <Grid item xs={12}>
+        <Typography variant="subtitle1">
+          {translate("checkout.payment.title")}
+        </Typography>
+      </Grid>
+      <Grid item xs={12}>
+        <Radio
+          name={translate("checkout.payment.payment-method.title")}
+          list={[
+            translate("checkout.payment.payment-method.e-money"),
+            translate("checkout.payment.payment-method.cash-on-delivery"),
```

Comment:
also the value selected from a radio button should not be language-dependent, we need to add value and label here